src/VisualStudio/Core/Def/CodeCleanup/AbstractCodeCleanUpFixer.cs

```diff
@@ -299,13 +293,12 @@ async Task<Solution> ApplyFixAsync(ProgressTracker progressTracker, Cancellation
                 cancellationToken.ThrowIfCancellationRequested();
 
                 var document = project.GetRequiredDocument(documentId);
-                progressTracker.Description = document.Name;
-
-                // FixDocumentAsync reports progress within a document, but we limit progress reporting for a project
-                // to the current document.
-                var documentProgressTracker = new ProgressTracker();
+                progressTracker.Report(CodeAnalysisProgress.Description(document.Name));
 
-                var fixedDocument = await FixDocumentAsync(document, enabledFixIds, documentProgressTracker, ideOptions, cancellationToken).ConfigureAwait(false);
+                // FixDocumentAsync reports progress within a document, but we only want to report progress at the
+                // project granularity.  So we pass CodeAnalysisProgress.None here so that inner progress updates don't
+                // affect us.
+                var fixedDocument = await FixDocumentAsync(document, enabledFixIds, CodeAnalysisProgress.None, ideOptions, cancellationToken).ConfigureAwait(false);
```

Comment:
a place in product code where we intentionally do not forward things along.  in these cases, we want to doc to make it clear this is intentional.